docs/legacy/blueprint-themes/blueprint-email-templates.md

```diff
@@ -20,7 +20,7 @@ Sent to the recipient of a gift certificate.
 | `%%LNG_GiftCertificateEmailInstructions%%` | "For instructions on how to redeem your gift certificate please <a href='%s/giftcertificates.php?action=redeem'>click here</a>." |
 | `%%LNG_GiftCertificateEmailAttached%%` | "Your gift certificate is attached to this email." |
 | `%%LNG_GiftCertificateEmailWarning%%` | "Please download or print a copy of your gift certificate for safe keeping as gift certificates are non-transferable." |
-| `%%GLOBAL_EmailFooter%%` | "<b>%s</b><br><a href="%s.mybigcommerce.com/">http://%s.mybigcommerce.com/</a><br>%s is powered by BigCommerce. <a href="bigcommerce.com">Launch your own store for free</a> with BigCommerce." |
+| `%%GLOBAL_EmailFooter%%` | "**%s**<br><a href="%s.mybigcommerce.com/">http://%s.mybigcommerce.com/</a><br>%s is powered by BigCommerce. <a href="bigcommerce.com">Launch your own store for free</a> with BigCommerce." |

```

Comment:
Not sure if the %s is supposed to show in the preview? It's like this throughout the document

Reply:
yes, %s is supposed to show. That's the character that represents the value of the store subdomain. Let's not mess with any of the html in this email templating table.